dwi_ml/experiment/monitoring.py

```diff
@@ -18,62 +20,75 @@ class LossHistoryMonitor(object):
 
     def __init__(self, name):
         self.name = name
-        self.history = []
-        self.epochs = []
-        self.sum = 0.0
-        self.count = 0
-        self._avg = 0.0
-        self.num_iter = 0
-        self.num_epochs = 0
+        self.history = []  # The list of loss values
```

Comment:
What do you think of renaming `history` as `updates`, and `current_history` as `current_epoch_updates`? I think it would help for clarity.